### ISSUE TITLE
Optional message after @ts-expect-error

### ISSUE THREAD

**Issue by jozef-mikusinec-kiwicom** [kawamataryo/suppress-ts-errors#460]
Hi!

I think it would be useful if I could do something like:

```bash
npx suppress-ts-errors --message="Flow to TS migration."
```

and get comments like these:

```ts
// @ts-expect-error: Flow to TS migration.

// @ts-expect-error TS7016: Flow to TS migration.
```

**Comment by kopach** [comment]
There is a great tool of similar kind, which can be used as inspiration here https://github.com/privatenumber/eslint-plugin-fix-later?tab=readme-ov-file#insertdisablecomment
It allows you to add message as well as insert `codeowner` in the comment, which is IMO great to delegate this fix
Also, there is an option to where to put to commend, after disable command or above

**Comment by jozef-mikusinec-kiwicom** [comment]
@kopach That looks great, but unfortunately it looks like that tool only works for ESLint errors, there is no mention of suppressing TypeScript errors

**Comment by kopach** [comment]
Yes, I've just referenced it here for implementation inspiration

**Comment by kawamataryo** [comment]
@jozef-mikusinec-kiwicom @kopach 

Thank you for your patience. I have added this option in version 1.5.0. Please give it a try.
